cdap-docs/admin-manual/source/installation/ambari.rst

```diff
@@ -349,6 +349,77 @@ not have a mechanism for setting the YARN ``allowed.system.users`` |---| the pre
 method of enabling the ``cdap`` user as it is more precise and limited |---| the setting
 of ``min.user.id`` needs to be used instead.)
 
+A. If you are **adding CDAP** to an existing Kerberos cluster, in order to configure **CDAP for
+   Kerberos authentication**:
+   
+   #. The ``<cdap-principal>`` is shown in the commands that follow as ``cdap``;
+      however, you are free to use a different appropriate name.
+
+      .. highlight:: console
+      
+   #. When running on a secure HBase cluster, as the ``hbase`` user, issue the command::
+
+        $ echo "grant 'cdap', 'RWCA'" | hbase shell
+
+   #. In order to configure **YARN for secure Hadoop:** the ``<cdap-principal>`` user must be 
+      able to launch YARN containers, either by adding it to the YARN ``allowed.system.users``
+      whitelist (preferred) or by adjusting the YARN ``min.user.id`` to include the ``<cdap-principal>`` user.
```

Comment:
This section is incorrect. The text above these changes related to `min.user.id` is accurate.